app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt

```diff
@@ -0,0 +1,232 @@
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kickstarter.R
+import com.kickstarter.ui.compose.designsystem.KSTheme
+import com.kickstarter.ui.compose.designsystem.KSTheme.colors
+import com.kickstarter.ui.compose.designsystem.KSTheme.dimensions
+import com.kickstarter.ui.compose.designsystem.KSTheme.typography
+
+enum class PaymentScheduleTestTags {
+    PAYMENT_SCHEDULE_TITLE,
+    DATE_TEXT,
+    AMOUNT_TEXT,
+    EXPAND_ICON,
+    BADGE_TEXT,
+    TERMS_OF_USE_TEXT,
+}
+
+enum class PaymentStatuses {
+    COLLECTED,
+    AUTHENTICATION_REQUIRED,
+    SCHEDULED
+}
+
+@Preview(
+    name = "Dark Collapsed State", uiMode = Configuration.UI_MODE_NIGHT_YES,
+)
+@Preview(
+    name = "Light Collapsed State", uiMode = Configuration.UI_MODE_NIGHT_NO,
+)
+@Composable
+fun PreviewCollapsedPaymentScheduleWhite() {
+    KSTheme {
+        PaymentSchedule(
+            isExpanded = false,
+            onExpandChange = {}
+        )
+    }
+}
+
+// Expanded State Preview
+@Preview(showBackground = true, name = "Expanded State")
+@Composable
+fun PreviewExpandedPaymentSchedule() {
+    KSTheme {
+        PaymentSchedule(
+            isExpanded = true,
+            onExpandChange = {}
+        )
+    }
+}
+
+@Preview
+@Composable
+fun InteractivePaymentSchedulePreview() {
+    var isExpanded by remember { mutableStateOf(false) }
+    KSTheme {
+        PaymentSchedule(
+            isExpanded = isExpanded,
+            onExpandChange = { isExpanded = it }
+        )
+    }
+}
+
+@Composable
+fun PaymentSchedule(
+    isExpanded: Boolean = false,
+    onExpandChange: (Boolean) -> Unit = {},
+    // TODO: Add payment schedule data model when available
```

Comment:
fyi, @jlplks we do create our own data models for the app. Once the API layer is ready (either GraphQL or REST) once the network call is made we will make the transformation required within the networking layer, take a look for example at  `KSApolloclient -> override fun getProject(slug: String): Observable<Project>`. The project returned is our own project -> `Project.kt`

Reply:
another fyi, if you want to know how the PaymentIncrement data model is gona look like on the API, take a look here -> https://kickstarter.atlassian.net/wiki/spaces/PAYM/pages/3210641443/PLOT+Mobile+API+Source+of+Truth?focusedCommentId=3212476418

Reply:
Created data model for it